crates/ere-hosts/src/main.rs

```diff
@@ -27,14 +31,39 @@ struct Cli {
     /// Action to perform
     #[arg(short, long, value_enum, default_value = "execute")]
     action: BenchmarkAction,
+
+    /// Source of blocks and witnesses
+    #[command(subcommand)]
+    source: SourceCommand,
+}
+
+#[derive(Subcommand, Clone, Debug)]
+enum SourceCommand {
+    Tests {
+        #[arg(short, long, default_value = "zkevm-fixtures/fixtures")]
+        directory_path: String,
+    },
+    Mainnet {
```

Comment:
BTW, I'm not convinced this is a good name, since it will be whatever network the corresponding RPC clients is synced to. Maybe `Rpc`? Or `External`?